src/app/clusters/dishwasher-alarm-server/dishwasher-alarm-server.cpp

```diff
@@ -0,0 +1,299 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+#include "dishwasher-alarm-server.h"
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/EventLogging.h>
+#include <app/InteractionModelEngine.h>
+#include <app/util/attribute-storage.h>
+#include <app/util/error-mapping.h>
+#include <lib/support/BitFlags.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::DishwasherAlarm;
+using namespace chip::app::Clusters::DishwasherAlarm::Attributes;
+using namespace chip::DeviceLayer;
+using chip::Protocols::InteractionModel::Status;
+
+using namespace std;
+
+DishwasherAlarmServer DishwasherAlarmServer::instance;
+
+DishwasherAlarmServer & DishwasherAlarmServer::Instance()
+{
+    return instance;
+}
+
+EmberAfStatus DishwasherAlarmServer::GetMaskValue(EndpointId endpoint, BitMask<AlarmMap> * mask)
+{
+    EmberAfStatus status = Attributes::Mask::Get(endpoint, mask);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        ChipLogProgress(Zcl, "Dishwasher Alarm: ERR: reading  mask, err:0x%x", status);
+        return status;
+    }
+
+    ChipLogProgress(Zcl, "Dishwasher Alarm: Mask ep%d value: %" PRIu32 "", endpoint, mask->Raw());
```

Comment:
Why is this back to PRIu32 for masks?  This sort of thing is pretty frustrating as a code reviewer, to be honest: If I can't trust that random things will not break I have to re-review the whole PR from the top every time instead of just looking at whether my comments were addressed.  :(

Reply:
Sorry, I do a double check now and fixes it.